packages/edit-site/src/components/sidebar/global-styles-v2/components/inspector.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * External dependencies
+ */
+import { motion } from 'framer-motion';
+import ReactJson from 'react-json-view';
+
+/**
+ * WordPress dependencies
+ */
+import {
+	Card,
+	CardBody,
+	CardHeader,
+	FormGroup,
+	Panel,
+	PanelBody,
+	PanelHeader,
+	Select,
+	useNavigatorHistory,
+	useNavigatorLocation,
+	View,
+} from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { useAppState } from '../state';
+
+const routes = [
+	'/',
+	'/colors',
+	'/colors/palette',
+	'/colors/elements/background',
+	'/typography',
+	'/typography/elements/headings',
+].map( ( o ) => ( {
+	value: o,
+	label: o,
+} ) );
+
+export const Inspector = () => {
```

Comment:
Should this be using the default sidebar inspector for blocks? Should it be named something else otherwise?

Reply:
Yep, I think we should name this otherwise to prevent from term confusion 😉